lib/nitcorn/package.ini

```diff
@@ -3,6 +3,7 @@ name=nitcorn
 tags=network,lib
 maintainer=Alexis Laferrière <alexis.laf@xymus.net>
 license=Apache-2.0
+more_contributors=Jean-Philippe Caissy <jean-philippe.caissy@shopify.com>, Guillaume Auger, Frederic Sevillano, Justin Michaud-Ouellette, Stephan Michaud, Maxime Bélanger
```

Comment:
more_contributors are for additional people not present in the git history

2015-11-07 11:00 GMT-05:00 Alexis Laferrière <notifications@github.com>:

> In lib/nitcorn/package.ini
> <https://github.com/nitlang/nit/pull/1815#discussion_r44213728>:
>
> > @@ -3,6 +3,7 @@ name=nitcorn
> >  tags=network,lib
> >  maintainer=Alexis Laferrière <alexis.laf@xymus.net>
> >  license=Apache-2.0
> > +more_contributors=Jean-Philippe Caissy <jean-philippe.caissy@shopify.com>, Guillaume Auger, Frederic Sevillano, Justin Michaud-Ouellette, Stephan Michaud, Maxime Bélanger
>
> @Morriar <https://github.com/Morriar> should probably be in this list,
> and @R4PaSs <https://github.com/R4PaSs>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nitlang/nit/pull/1815/files#r44213728>.
>